tools/rosparam/src/rosparam/__init__.py

```diff
@@ -160,11 +160,16 @@ def load_file(filename, default_namespace=None, verbose=False):
       corresponding namespaces for each YAML document in the file
     :raises: :exc:`RosParamException`: if unable to load contents of filename
     """
-    if not os.path.isfile(filename):
-        raise RosParamException("file [%s] does not exist"%filename)
-    if verbose:
-        print("reading parameters from [%s]"%filename)
-    f = open(filename, 'r')
+    if filename == '-' or not len(filename):
```

Comment:
You can use `not filename` directly without the need to call `len()`.

Same below.